docs/shop_administration/product_categories.md

```diff
@@ -0,0 +1,92 @@
+---
+description: You can create a structure of tags that can be assigned to products to help categorize products, to assist users in searching. One product can belong to different categories. In 4.2 you create, manage and assign categories manually, no import is available. In future releases categories will be assigned automatically based on rules, export/import will be added, as well as rules for applying different sales conditions based on categories.
```

Comment:
Let's not pin down version numbers in the description, otherwise you'd have to update this with every release. Also, description should ideally be kept to no more than 160 characters